php/libraries/Candidate.class.inc

```diff
@@ -729,6 +729,13 @@ class Candidate
                 $pscid = Candidate::generateRandomStringFromArrays($pscidArrays);
             }
 
+            if ($pscid == '') {
```

Comment:
@jstirling correct me if i'm wrong but is this ever reached with the fix you made below ?? is it a safety check?